main/tests.py

```diff
@@ -77,3 +78,28 @@ def test_dog_owner_homepage_is_visible_for_dog_owner(self, client, create_dog_ow
         response = client.get("/homepage/")
         assert response.status_code == 200
         assert list(response.context['daycares']) == list(DayCare.objects.all())
+
+
+@pytest.mark.django_db
+class TestOrdersView:
+    def test_orders_page_is_visible_for_dog_owner_user(self, client, create_dog_owner_user):
+        client.force_login(user=create_dog_owner_user.user)
+        response = client.get("/orders/")
+        assert response.status_code == 200
+
+    def test_orders_page_is_visible_for_daycare_user(self, client, create_daycare_user):
+        client.force_login(user=create_daycare_user.user)
+        response = client.get("/orders/")
+        assert response.status_code == 200
+
+    def test_only_relevant_orders_are_displayed_for_daycare(self, client, create_daycare_user):
+        client.force_login(user=create_daycare_user.user)
+        response = client.get("/orders/")
+        assert list(response.context['orders']) == list(Order.objects.filter(daycare_id=create_daycare_user))
```

Comment:
Please take a look at https://github.com/redhat-beyond/Hotails/pull/64#discussion_r862467315 about naming variabels in assert

Reply:
Changed.